idx/cassandra/cassandra.go

```diff
@@ -266,8 +266,14 @@ func (c *CasIdx) AddOrUpdate(data *schema.MetricData, partition int32) {
 	}
 
 	if updateIdx {
+		// set the new metricdef based on the data, however if this data will go back in time
+		// it'll be rejected data, and we should stick to the higher LastUpdate we already had.
 		def := schema.MetricDefinitionFromMetricData(data)
+		if inMemory && existing.LastUpdate > def.LastUpdate {
```

Comment:
existing.LastUpdate will be greater then def.LastUpdate when processing a backlog at startup.  We need to accept those metrics to refill the aggMetric chunks